src/quota_aggregator_impl.cc

```diff
@@ -137,6 +142,18 @@ ::google::protobuf::util::Status QuotaAggregatorImpl::Quota(
   // Aggregate tokens if the cached response is positive
   if (lookup.value()->is_positive_response()) {
     lookup.value()->Aggregate(request);
+  } else {
+    if (lookup.value()->in_flight() == false) {
+      int64_t elapsed = (SimpleCycleTimer::Now() - lookup.value()->last_refresh_time())
```

Comment:
can we create a refresh_interval_in_cycle_ variable, just like

https://github.com/cloudendpoints/service-control-client-cxx/blob/master/src/check_aggregator_impl.cc#L65

Then compare the time stamp like

https://github.com/cloudendpoints/service-control-client-cxx/blob/master/src/check_aggregator_impl.cc#L166